src/components/TeacherDashboard/DashboardLayout.js

```diff
@@ -1,12 +1,14 @@
 import React from 'react';
 import { Image, Menu, Icon } from 'semantic-ui-react';
 import { Navigate, Link, Outlet } from 'react-router-dom';
+import AuthService from '../../services/auth.service';
 
 function DashboardLayout() {
     const logOut = () => {
         // TODO: implement sign out process
         // fetch to backend authenticate
         // if authenticated
+        AuthService.logout();
         console.log('Sign Out');
         Navigate('/teachers');
```

Comment:
Change Navigate('/teachers') to:
```
const navigate = useNavigate();
navigate('/login', { replace: true});
```

Setting replace: true won't create another entry in the history from logging out so they can't go back to the protected routes once they get redirected to the login page